src/components/EditCoursePage/AdditionalMetadataFields.jsx

```diff
@@ -44,6 +44,13 @@ function AdditionalMetadataFields(props) {
           disabled={disabled}
           required
         />
+        <Field
+          name="additional_metadata.variant_id"
+          component={RenderInputTextField}
+          label=<FieldLabel id="variant_id.label" text="Course Variant Id" />
+          disabled={disabled}
+          required={false}
```

Comment:
nit: If `required` isn't passed, it's implicitly not required, so this line is technically unnecessary, though can't hurt to be explicit I guess 🤷‍♂️

Reply:
+1 to this.